### ISSUE TITLE
Regency

### ISSUE THREAD

**Issue by anhegofcherek** [prophecymud/ideas#113]
An idea I've been kicking around in various forms, to add some asynchronous nonlethal PvP, capitalise on the ease of making changes etc. - see also #103 etc.

Roughly:

* Players can become "regents" of an area (a little like hometowns)
* Become a regent by killing some sort of not-easy-but-not-too-hard mob ("The Spirit of the Prophecy" or something, to loosely fit it in story-wise), taking the regency from the previous regent
* Regents get periodic rewards (gold? pkpts? pretitle/lastname?) during their regency

Open questions include:

* What should the rewards be?
* PK-only or not?
* If not PK-only, how to deal with potential (probable?) killstealing

It gets people out of their halls, and it gives a form of asynchronous competition. PK-wise, rewards are fairly simple - pkpoints (#104) fit nicely, and can be given out relatively casually - more thought would be needed for non-PK rewards.

If a temporary pretitle / lastname for the duration, then I'd probably need to flesh out the code for those to let them be selectable / temporary / something, but that should be relatively straightforward.

Thoughts?

**Comment by grekan** [comment]
Love the idea. I would obviously like to see it available to non-pk. Perhaps RPPTs as a reward for non-pk?

The temporarily pretitle/last name is probably less desirable. Perhaps some way to purchase permanent versions with the points? 

Make the room containing the spirit private and there's no kill stealing?

**Comment by anhegofcherek** [comment]
> Perhaps RPPTs as a reward for non-pk?

You're not RPing though - I'd like to keep those as designed

> Make the room containing the spirit private and there's no kill stealing?

Hampers the PK aspect alas

**Comment by grekan** [comment]
I might consider inventing a new point just for this tbh. 

Don't worry about kill stealing? Let the mortals work it out? Or some kind of mprog that once it's been engaged locks it to that person for 5 min?

**Comment by kerowa** [comment]
What should the rewards be? 

Some form of pkpts or scrimpts perhaps?  Since it involves some form of "fighting" albeit against a mob.

PK-only or not?

Would like to see it be available to both, but perhaps the rewards could be higher for PK.

If not PK-only, how to deal with potential (probable?) killstealing

Well there's always the option where killstealing is not permitted for anyone per help_Standardpenalties:p

**Comment by anhegofcherek** [comment]
> Don't worry about kill stealing? Let the mortals work it out?

I'd still need to actively do something, like declare that killstealing was ok in that instance

**Comment by anhegofcherek** [comment]
> Some form of pkpts or scrimpts perhaps? Since it involves some form of "fighting" albeit against a mob.

Yeah piggybacking on scrimpts feels maybe ok, but I'm a bit wary of automated ways to get (effectively) qp - the advantage/hope with pkpts is that at least by keeping that in PK, there's more flow / loss / etc.

> Would like to see it be available to both, but perhaps the rewards could be higher for PK.

Yeah if it's for both, I'd definitely make it higher for PK

> Well there's always the option where killstealing is not permitted for anyone per help_Standardpenalties:p

Yep, it just feels a bit ridiculous if I'm the Regent of X, I see someone coming along and trying to claim it from me, and all I can do is stand there and watch and wait and try to reclaim afterwards

With PK you can defend your regency with combat. For non-PK...I think there needs to be another mechanism

**Comment by kerowa** [comment]
I think kill stealing should be allowed in the specific case of regency for the reason that you would want to defend your regency -- in practice, I don't think it would happen as often as expected.    

Ok - that's fair.   Could there be a regencypt system?  Where you see how it's working out and imms can do some sort of reasonable conversion?

Random idea, the mob gives an area shout when it's attacked.  Perhaps the regency should have a regent's room in the area ala the clan hall room, so you can stay inthe area ?

**Comment by anhegofcherek** [comment]
> I think kill stealing should be allowed in the specific case of regency for the reason that you would want to defend your regency

Fair but even then it doesn't feel great - so what, we both stand there attacking the same mob, hoping that one of us is the lucky one who lands the killing blow? That just feels like a recipe for frustration

**Comment by anhegofcherek** [comment]
Hm maybe I could work something in with an automated PKQuest for nonpkers - that way people could attack each other...

**Comment by CorlanProphecy** [comment]
Well certainly pk and non-pk need to have separate systems atleast. MY suggestions would be to make some areas non-pk and others pk. Some areas are more popular among pkers because of their transportation-dyanmics like norecall/no gate, for instance Rak Cthol, Drojim palace, Kotu, probably some others but i havent pked in over 10 years so I cant recall atm. Separate areas would also make the amount of areas to modify more reasonable.

For pk areas the mechanism of killing either a local mob or the current regent would pass on regency, regency needs to decay if the regent doesnt log on. I would also suggest for pk regency the regent and the regents clan would recieve rewards to encourage clan members to defend their clans local interests.

Rewards is tricky and would require balancing, i like the idea of flairs/pre-titles (a PK regent could be Enforcer, also something like Tax-collector...) providing prestige coupled with lower amount rewards like Aqps would be an excellent reward for pkers because aqing is harder, pkpts for making qeq is another option, id rather have that than gold because its more desirable.

**Comment by anhegofcherek** [comment]
> MY suggestions would be to make some areas non-pk and others pk.

I like it

> For pk areas the mechanism of killing either a local mob or the current regent would pass on regency

Hm, I think maybe just the mob - I don't want to impose penalties for people dying in PK

> regency needs to decay if the regent doesnt log on.

I agree with the principle but I don't think this needs to be explicit

If you don't log on, you're not around to defend your regency and someone else can take it - problem solved

>  I would also suggest for pk regency the regent and the regents clan would recieve rewards to encourage clan members to defend their clans local interests.

Agreed. Mechanics might be a bit fiddly but this is my mid/long-term goal - not just individual conflict, but clan-level too

> Rewards is tricky and would require balancing

Yup....! Good idea re aqp

**Comment by prophnoran** [comment]
Cool idea; as for rewards, how about just a leaderboard/bragging rights for now? I think that would encourage people to try it out, and we can get the kinks out before trying to figure out what it's worth.

Kill stealing is a challenging question. Normally we want to disallow it, but here it seems a key way to defend your regency. So I suggest this: don't allow kill stealing here (for now, honor system), but if you lose a regency, you have 48h to kill the mob again, in which case it's as though you never lost it. So you can 'defend' your regency without kill-stealing.

One additional idea, which I think would be appropriate down-the-line but not needed for an initial try: as written there is a sort of game theoretic instability, where any two players who can kill the mob are likely to just go back and forth forever. Maybe that's okay, but...  to ameliorate, you can record the time it takes to kill the mob. To steal the regency, you have to beat the current regent's time (or vary the difficulty some other way). The challenge level should slowly degrade, though.

**Comment by prophnoran** [comment]
One other down-the-line idea: this would be a cool way to experiment with mob AI.
